### PR TITLE
Updated examples on the homepage, customized page titles

### LINKED ISSUE #29: front page has non working example
Screen Shot 2022-06-02 at 06 33 52

As we know `scoop install vscode` will actually fail because there's no mention of the `extras` bucket. However, rather than removing the example, I think there could be a few. I just found out about the `versions` bucket myself which wasn't obvious. Maybe provide some examples with each and mention buckets. 

Either way, at the very least, this example needs to be fixed.

### PR DESCRIPTION
**Pull request by gpailler**
- Fixed #29 with 2 working examples (7zip and vscode)
- Customized titles for a better browser history
- Better highlighting for matches

### CODE REVIEW

**Comment by gpailler** [comment]
@rashil2000 Feel free to approve the merge if it looks good to you https://scoop.sh/scoopinstaller-dev.github.io/

File: src/components/Home.tsx [hunk 1]
```diff
@@ -103,9 +103,30 @@ const Home = (): JSX.Element => {
...
+Persisting Formats
+Running post_install script...
+'7zip' (21.07) was installed successfully!`}
+                </SyntaxHighlighter>
+              </Tab>
+              <Tab eventKey="vscode" title="VS Code">
+                <SyntaxHighlighter language="json">
+                  {`> scoop bucket add extras
+Checking repo... OK
+The versions bucket was added successfully.
```
Comment (rashil2000) [inline src/components/Home.tsx:127]:
```suggestion
The extras bucket was added successfully.
```

**Comment by rashil2000** [comment]
I was planning replace to replace vscode with neovim (because it's in _Main_), but this idea is better! 

I think we should do neovim + vscode, because 7zip is an internal dependency of Scoop and usually gets installed automatically.

File: src/components/Home.tsx (hunk 1 above)
Comment (gpailler) [reply src/components/Home.tsx:127]: Oh nice catch 👍

**Comment by gpailler** [comment]
I was looking for a mainstream package in main but you are right, 7zip is probably not the best choice.
What about python then? I think it's one of the most popular package we can imagine

**Comment by rashil2000** [comment]
Python iw already used as example in the next paragraph 😅

**Comment by gpailler** [comment]
Yes, and it show a continuity in the examples, don't you think?
Do you have any other mainstream package in mind?

**Comment by rashil2000** [comment]
The example should have both `bin` and `shortcuts`, should be a popular one and be present in the Main bucket 😅

Apart from Neovim, there's DOSBox and PostgreSQL that I can think of...

**Comment by gpailler** [comment]
What a challenge 😄 ! 
I think we should focus on the most popular programs for a first 'wow' effect, not sure the `bin` and `shortcuts` are that important in theses examples.

What about adding even more examples ?
K6PG2BDKuX

**Comment by sitiom** [comment]
Perhaps you can display those as embed [asciinema](https://asciinema.org/) recordings for some interactivity. What do you think?

**Comment by rashil2000** [comment]
> What about adding even more examples ?

I like this idea. But I still think Python can be skipped (Node.js is similar to it, and also 4 examples are enough 😅)

**Comment by gpailler** [comment]
Done. Python is removed and replaced with Node.js as a first example

**Comment by rashil2000** [comment]
Great!

I've pushed a commit to update search command output in accordance with https://github.com/ScoopInstaller/Scoop/pull/4997

**Comment by gpailler** [comment]
Looks good! Feel free to merge then

**Pull request merged**